docs/data/introduction/support/support.md

```diff
@@ -34,18 +43,25 @@ Including your Order ID (or Support key) in the issue helps us prioritize the is
 2. MUI X Premium: The same as MUI X Pro, but with higher priority.
 3. MUI X Priority Support add-on (coming soon): Provides a 24h SLA for the first answer.
 
-### Long-term support (LTS)
+## Long-term support (LTS)
 
-Bug fixes, performance enhancements, and other improvements are delivered in new releases. However, MUI remains committed to providing security updates and addressing regressions for the immediate predecessor of the current major version.
+Bug fixes, performance enhancements, and other improvements are delivered in new releases. However, we remain committed to providing security updates and addressing regressions for the immediate predecessor of the current major version.
 
 This includes issues introduced by external sources, like browser upgrades or changes to upstream dependencies.
 
 ### Supported versions
 
 - MUI X v7: ✅ Pre-release (Continuous support - Stable release: March 2024)
 - MUI X v6: ✅ Stable major (Continuous support)
-- MUI X v5: ⚠️ Long term support (Guaranteed Support for security issues and regressions).
+- MUI X v5: ⚠️ Long-term support (Guaranteed Support for security issues and regressions).
 - MUI X v4: 🅧 No longer supported.
 - MUI X v3: 🅧 Never existed.
 - MUI X v2: 🅧 Never existed.
 - MUI X v1: 🅧 Never existed.
+
+## Community
+
+We have a [Discord Server](https://mui.com/r/discord/) to allow the MUI X community to gather.
+Our tools are used by thousands of developers and teams all around the world, many of whom actively engage with the community.
+
+You can join Discord to engage in lively discussions, share your projects, and interact with developers just like you from all around the world. We'd love for you to be a part of our community!
```

Comment:
It feels like we can still link to Discord, but further buried and with a clear purpose statement.